src/gasPriceOracle/adapters/ethereum.ts

```diff
@@ -29,7 +37,7 @@ export async function eip1559Raw(provider: providers.Provider, chainId: number):
   assert(BigNumber.isBigNumber(baseFeePerGas), `No baseFeePerGas received on ${getNetworkName(chainId)}`);
 
   return {
-    maxFeePerGas: maxPriorityFeePerGas.add(baseFeePerGas),
+    maxFeePerGas: maxPriorityFeePerGas.add(baseFeePerGas).mul(baseFeeMultiplier),
```

Comment:
@pxrl i fixed this in your PR where you were applying the `.mul(baseFeeMultiplier)` to the `maxPriorityFeePerGas` rather than the  `baseFeePerGas` which I believe was a mistake. LMK

Reply:
Agree - good catch, thanks!